src/gui/src/layoutViewer.cpp

```diff
@@ -2049,6 +2064,7 @@ void LayoutViewer::drawBlock(QPainter* painter,
                                      bounds.yMax(),
                                      instance_limit);
 
+    painter->setRenderHint(QPainter::RenderHint::Antialiasing, false);
```

Comment:
Shapes on the RDL layer often contain 45 degree segments and would not be anti-aliased.